From: Dima Kogan <dima@secretsauce.net>
Date: Sat, 17 Nov 2012 23:58:48 -0800
Subject: moved NEWS blurb from a later committer back.

Upstream forgot to update the NEWS before cutting the tag
---
 NEWS |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/NEWS b/NEWS
index 4c3a15d..d4b2a7a 100644
--- a/NEWS
+++ b/NEWS
@@ -1,3 +1,10 @@
+Version 1.3.0 SEP 30 2012 
+
+Release for end of FY2012, includes bug fixes, better support for
+autoconf, DFXML standardizations, and the ability to compile under
+mingw for Windows (that was a LOT of work).
+
+
 Version 1.2.7 May 24 2012 (GIT)
 
 I am pleased to announce the release of tcpflow version 1.2.7
